cinema/models.py

```diff
@@ -2,6 +2,20 @@
 from django.db import models
 from django.conf import settings
 
+import os
+import pathlib
+import uuid
+
+from django.utils.text import slugify
+
+
+def create_custom_path(instance, filename):
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
+    return pathlib.Path("uploads/movies/") / pathlib.Path(filename)
```

Comment:
you don't need to use `pathlib.Path(filename)`
It can be simplified to `pathlib.Path("uploads/movies/") / filename`